src-vue/src/views/mods/LocalModsView.vue

```diff
@@ -14,6 +14,7 @@
                     </template>
                 </el-popconfirm>
                 {{ mod.name }}
+                (v{{ mod.version }})
```

Comment:
Given you've defined version type as `Option<String>`, what happens here if `mod.version` is null?

Reply:
Just turns into nothing it seems.
![image](https://user-images.githubusercontent.com/40122905/217049179-6426df27-9520-4fdc-b42b-d434445243cc.png)

Given that it is such a rare occasion should we add edge case handling (I think Northstar prints `0.0.0` as the version number then) or just ignore?

Reply:
If this happens, you shouldn't display version indicator at all.

Reply:
Aight, done with 5274aa9ada22d9299e87fb41c4d42214517a97fd